src/card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-concrete);
+}
+  
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-left: calc(50% - 160px / 2);
```

Comment:
Image should be centered, don't add fixed margin-left for the image
Just add correct paddings to the main card